Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -40,11 +41,34 @@ import PayPalCheckout
         tokenize(request: request, completion: completion)
     }
 
+    /// Tokenize a PayPal request to be used with the PayPal Native Checkout flow.
+    ///
+    /// On success, you will receive an instance of `BTPayPalNativeCheckoutAccountNonce`.
+    /// On failure or user cancelation you will receive an error. If the user cancels
+    /// out of the flow, the error code will equal `BTPayPalNativeError.canceled.rawValue`.
+    ///
+    /// - Parameter request: A `BTPayPalNativeCheckoutRequest`
+    /// - Returns: A `BTPayPalNativeCheckoutAccountNonce` if successful
+    /// - Throws: An `Error` describing the failure
+    @objc(tokenizeWithNativeCheckoutRequest:completionHandler:)
+    public func tokenize(_ request: BTPayPalNativeCheckoutRequest) async throws -> BTPayPalNativeCheckoutAccountNonce {
```

Comment:
Same thing here with future testing being captured in DTBTSDK-2512 - we currently only test 1 thing (an error) in BTPayPalNativeCheckoutClient_Tests. 😱 Testing this may be harder/impossible due to not being able to easily mock NXO methods, so this may be the best we can do, but worth revisiting.

Also tested this in the demo app though to ensure it's working as expected since we can't easily unit test this.

Reply:
Yeh, our testing for PP Native modules on BT and PPCP is really limited - mostly due to NXO limitations (no @testable import support, and all classes are `final`). 

We can punt for now, but at some point should outline what exactly we want from them in terms of testing capability.